src/relayclient/RelayClient.ts

```diff
@@ -157,8 +158,14 @@ export default class RelayClient {
     const { relayRequest, approvalData, signature, httpRequest } =
       await this._prepareRelayHttpRequest(relayInfo, gsnTransactionDetails)
     const acceptRelayCallResult = await this.contractInteractor.validateAcceptRelayCall(relayRequest, signature, approvalData)
-    if (!acceptRelayCallResult.success) {
-      return { error: new Error(`canRelay failed: ${acceptRelayCallResult.returnValue}`) }
+    if (!acceptRelayCallResult.paymasterAccepted) {
+      let message: string
+      if (acceptRelayCallResult.reverted) {
+        message = 'local view call to \'relayCall()\' reverted'
+      } else {
+        message = 'paymaster rejected in local view call to \'relayCall()\' '
```

Comment:
lost reject reason... should report it as well.

Reply:
no I didn't, look literally 1 line below